src/cli/init/utils.test.ts

```diff
@@ -1,28 +1,20 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-FileCopyrightText: 2023-Present The Pepr Authors
 
-import { expect, test } from "@jest/globals";
+import { expect, it } from "@jest/globals";
 
 import { sanitizeName } from "./utils";
 
-test("sanitizeName() sanitizes names correctly", () => {
-  const cases = [
-    {
-      input: "My Test Module",
-      expected: "my-test-module",
-    },
-    {
-      input: "!! 123 @@ Module",
-      expected: "123-module",
-    },
-    {
-      input: "---Test-Module---",
-      expected: "test-module",
-    },
-  ];
 
-  for (const { input, expected } of cases) {
-    const result = sanitizeName(input);
-    expect(result).toBe(expected);
-  }
-});
+it.each([
+  //Test sanitizeName() with ["$UNSANITIZED_INPUT", "$SANITIZED_INPUT"]
+  ["My Test Module", "my-test-module"],
+  ["!! 123 @@ Module", "123-module"],
+  ["---Test-Module---", "test-module"]
+])("sanitizeName() sanizes '%s' correctly", (input: string, expected: string) => {
+  expect(sanitizeName(input)).toBe(expected)
+})
+
+it("sanitizeName() should throw TypeError when given a non-string", () =>{
+  expect(() => sanitizeName({input: 0} as unknown as string)).toThrow(TypeError);
```

Comment:
This test case is needed since `sanitizeName()` processes untrusted CLI input provided from `walkthrough() -> setName()`